admin/tests/unit/models/certification-details_test.js

```diff
@@ -27,33 +27,73 @@ module('Unit | Model | certification details', function (hooks) {
   });
 
   module('#get competences', function () {
-    test('returns competences', function (assert) {
-      // given
-      const store = this.owner.lookup('service:store');
-      const listChallengesAndAnswers = [
-        { id: 'answerId1', competence: '1.1' },
-        { id: 'answerId2', competence: '1.1' },
-        { id: 'answerId3', competence: '1.2' },
-      ];
-      const competencesWithMark = [{ index: '1.1' }, { index: '1.2' }];
-
-      // when
-      const certification = store.createRecord('certification-details', {
-        listChallengesAndAnswers,
-        competencesWithMark,
+    module('when there are Pix+ competences', function () {
+      test('returns competences with Pix+ competences included', function (assert) {
+        // given
+        const PIX_PLUS_INDEX = '';
+        const store = this.owner.lookup('service:store');
+        const listChallengesAndAnswers = [
+          { id: 'answerIdPix+1', competence: PIX_PLUS_INDEX },
+          { id: 'answerIdPix+2', competence: PIX_PLUS_INDEX },
+          { id: 'answerId2', competence: '1.1' },
+          { id: 'answerId3', competence: '1.2' },
+        ];
+        const competencesWithMark = [{ index: '1.1' }, { index: '1.2' }];
+
+        // when
+        const certification = store.createRecord('certification-details', {
+          listChallengesAndAnswers,
+          competencesWithMark,
+        });
+
+        // then
+        assert.deepEqual(certification.competences, [
+          {
+            index: PIX_PLUS_INDEX,
+            name: 'Pix +',
+            answers: [
+              { id: 'answerIdPix+1', competence: '', order: 1 },
+              { id: 'answerIdPix+2', competence: '', order: 2 },
+            ],
+          },
+          {
+            index: '1.1',
+            answers: [{ id: 'answerId2', competence: '1.1', order: 3 }],
+          },
+          { index: '1.2', answers: [{ id: 'answerId3', competence: '1.2', order: 4 }] },
+        ]);
       });
+    });
 
-      // then
-      assert.deepEqual(certification.competences, [
-        {
-          index: '1.1',
-          answers: [
-            { id: 'answerId1', competence: '1.1', order: 1 },
-            { id: 'answerId2', competence: '1.1', order: 2 },
-          ],
-        },
-        { index: '1.2', answers: [{ id: 'answerId3', competence: '1.2', order: 3 }] },
-      ]);
+    module('when there are only Pix competences', function () {
```

Comment:
est-ce qu'on préciserait pas "core" ? 
```suggestion
    module('when there are only Pix core competences', function () {
```